ec2/README.md

```diff
@@ -0,0 +1,69 @@
+# CATalogASB EC-2 Deployment
+
+catasb is a collection of playbooks to create an OpenShift environment with a Service Catalog & Ansible Service Broker in EC-2.
+
+### Overview
+These playbooks will:
+  * Create a public VPC if it does not exist
+  * Create a security group if it does not exist
+  * Create an instance with a specific Name if does not exist
+  * Associate an elastic ip to the instance
+  * Configure a hostname with the elastic ip through Route53
+  * Setup Origin through `oc cluster up`
+  * <WIP> Install Service Catalog on Origin
+  * <WIP> Install Ansible Service Broker on Origin
+
+### Pre-Reqs
+  * Ansible needs to be installed so its source code is available to Python.
+    * Check to see if Ansible modules are available to Python
+            $ python -c "import ansible;print(ansible.__version__)"
+            2.2.2.0
+    * MacOS requires Ansible to be installed from `pip` and not `brew`
+          $ python -c "import ansible;print(ansible.__version__)"
+          Traceback (most recent call last):
+          File "<string>", line 1, in <module>
+          ImportError: No module named ansible
+
+          brew uninstall ansible
+          pip install ansible
+
+          $ python -c "import ansible;print(ansible.__version__)"
+          2.2.2.0
+  * Install python dependencies
+     * `pip install boto boto3 six`
```

Comment:
Let's add a note this is needed for python2.

Steve saw errors on his system where he was running python3.
He had to do a pip2 install to get the deps in the right spot for ansible.